File: dal-dao-gen/dal_ftp/63/cs/Entity/AbacusGetTaxLogGen.cs
using System;
using Arch.Data.Orm;

namespace com.ctrip.dal.test.test4.Entity.DataModel
{
    /// <summary>
    /// AbacusGetTaxLog
    /// </summary>
    [Serializable]
    [Table(Name = "AbacusGetTaxLog")]
    public partial class AbacusGetTaxLogGen
    {
        /// <summary>
        /// 获取或设置LogID
        /// </summary>
        [Column(Name = "LogID",Length=10),ID,PK]
        public int LogID { get; set; }
        /// <summary>
        /// 获取或设置UID
        /// </summary>
        [Column(Name = "UID",Length=50)]
        public string UID { get; set; }
        /// <summary>
        /// 获取或设置OrderID
        /// </summary>
        [Column(Name = "OrderID",Length=10)]
        public int? OrderID { get; set; }
        /// <summary>
        /// 获取或设置PNR
        /// </summary>
        [Column(Name = "PNR",Length=10)]
        public string PNR { get; set; }
        /// <summary>
        /// 获取或设置BeginTime
        /// </summary>
        [Column(Name = "BeginTime",Length=23)]
        public DateTime BeginTime { get; set; }
        /// <summary>
        /// 获取或设置EndTime
        /// </summary>
        [Column(Name = "EndTime",Length=23)]
        public DateTime EndTime { get; set; }
        /// <summary>
        /// 获取或设置Result
        /// </summary>
        [Column(Name = "Result",Length=10)]
        public int Result { get; set; }
        /// <summary>
        /// 获取或设置Tax
        /// </summary>
        [Column(Name = "Tax",Length=10)]
        public decimal? Tax { get; set; }
        /// <summary>
        /// 获取或设置ErrDesc
        /// </summary>
        [Column(Name = "ErrDesc",Length=2000)]
        public string ErrDesc { get; set; }
        /// <summary>
        /// 获取或设置Source
        /// </summary>
        [Column(Name = "Source",Length=20)]
        public string Source { get; set; }
        /// <summary>
        /// 获取或设置InsertTime
        /// </summary>
        [Column(Name = "InsertTime",Length=23)]
        public DateTime? InsertTime { get; set; }
        /// <summary>
        /// 获取或设置PassengerType
        /// </summary>
        [Column(Name = "PassengerType",Length=5)]
        public string PassengerType { get; set; }
        /// <summary>
        /// 获取或设置OperatingAirline
        /// </summary>
        [Column(Name = "OperatingAirline",Length=50)]
        public string OperatingAirline { get; set; }
        /// <summary>
        /// 获取或设置DataChangeLastTime
        /// </summary>
        [Column(Name = "DataChangeLastTime",Length=23)]
        public DateTime? DataChangeLastTime { get; set; }
    }
}

File: dal-dao-gen/dal_ftp/63/java/Entity/AbacusGetTaxLogGen.java
package com.ctrip.dal.test.test4;

import java.math.BigDecimal;
import java.sql.Timestamp;

import com.ctrip.platform.dal.dao.DalPojo;

public class AbacusGetTaxLogGen implements DalPojo {
	private Integer logID;
	private String uID;
	private Integer orderID;
	private String pNR;
	private Timestamp beginTime;
	private Timestamp endTime;
	private Integer result;
	private BigDecimal tax;
	private String errDesc;
	private String source;
	private Timestamp insertTime;
	private String passengerType;
	private String operatingAirline;
	private Timestamp datachangeLasttime;
	public Integer getLogID() {
		return logID;
	}

	public void setLogID(Integer logID) {
		this.logID = logID;
	}

	public String getUID() {
		return uID;
	}

	public void setUID(String uID) {
		this.uID = uID;
	}

	public Integer getOrderID() {
		return orderID;
	}

	public void setOrderID(Integer orderID) {
		this.orderID = orderID;
	}

	public String getPNR() {
		return pNR;
	}

	public void setPNR(String pNR) {
		this.pNR = pNR;
	}

	public Timestamp getBeginTime() {
		return beginTime;
	}

	public void setBeginTime(Timestamp beginTime) {
		this.beginTime = beginTime;
	}

	public Timestamp getEndTime() {
		return endTime;
	}

	public void setEndTime(Timestamp endTime) {
		this.endTime = endTime;
	}

	public Integer getResult() {
		return result;
	}

	public void setResult(Integer result) {
		this.result = result;
	}

	public BigDecimal getTax() {
		return tax;
	}

	public void setTax(BigDecimal tax) {
		this.tax = tax;
	}

	public String getErrDesc() {
		return errDesc;
	}

	public void setErrDesc(String errDesc) {
		this.errDesc = errDesc;
	}

	public String getSource() {
		return source;
	}

	public void setSource(String source) {
		this.source = source;
	}

	public Timestamp getInsertTime() {
		return insertTime;
	}

	public void setInsertTime(Timestamp insertTime) {
		this.insertTime = insertTime;
	}

	public String getPassengerType() {
		return passengerType;
	}

	public void setPassengerType(String passengerType) {
		this.passengerType = passengerType;
	}

	public String getOperatingAirline() {
		return operatingAirline;
	}

	public void setOperatingAirline(String operatingAirline) {
		this.operatingAirline = operatingAirline;
	}

	public Timestamp getDatachangeLasttime() {
		return datachangeLasttime;
	}

	public void setDatachangeLasttime(Timestamp datachangeLasttime) {
		this.datachangeLasttime = datachangeLasttime;
	}

}